pysmurf/tune/smurf_tune.py

```diff
@@ -689,27 +696,29 @@ def find_peak(self, freq, resp, rolling_med=True, window=5000,
         if timestamp is None:
             timestamp = self.get_timestamp()
 
+        # Break apart the data
         angle = np.unwrap(np.angle(resp))
         x = np.arange(len(angle))
         p1 = np.poly1d(np.polyfit(x, angle, 1))
         angle -= p1(x)
-        grad = np.convolve(angle, np.repeat([1,-1], 8),
-                           mode='same')
-        #grad = np.ediff1d(angle, to_end=[np.nan])
+        grad = np.convolve(angle, np.repeat([1,-1], 8), mode='same')
```

Comment:
@eyyoung24 should the 8 hardcode here also be made a function argument?  Too lazy to look up np.convolve.